operations/model.go

```diff
@@ -155,9 +159,50 @@ func (s *ClientSettings) setupRestCommunicator(ctx context.Context, printMessage
 	if printMessages {
 		printUserMessages(ctx, c, !s.AutoUpgradeCLI)
 	}
+
+	if s.shouldGenerateJWT(ctx, c) {
+		grip.Info("We will attempt to generate a JWT token, to opt out of this, set 'do_not_run_kanopy_oidc' to true in your config file")
+		if s.JWT, err = RunKanopyOIDCLogin(); err != nil {
```

Comment:
nit: should this function be exported, or is it only used within this package? If it's only used in this one package, it should be `runKanopyOIDCLogin`.